railties/lib/rails/templates/rails/welcome/index.html.erb

```diff
@@ -82,6 +86,10 @@
     </a>
   </nav>
 
+  <section>
+    <p>Edit your Rails app code at <code><%= Rails.root %></code> and build something amazing.</p>
```

Comment:
How is this information useful given the person already started the server and it is already in the folder? Don’t they already know where the code is?

Reply:
Another option is to link to the getting started guide instead. I could see the folder where the app is defined being useful if we have some kind of “one click installer” that would generate an app to you and start a server, but so far I’m not aware of any, neither I think we should optimize this page for this case.

Reply:
Sending folks to guides feels like it gives them too many "choices" and would "throw the book at them"

At this point I think the mindset is, "how do I make this screen go away?". Some ideas I had around that:

> "Edit the `Rails.root.join("config/routes.rb")` routes file and connect this application to a controller"

The language isn't perfect there, but better than nothing.

The developer would open `config/routes.rb` and maybe there's a comment in there telling them to generate a resource of scaffolding if its their first go at a Rails app.

Reply:
I like that better. Can you update the pr to point to the routes please?

Reply:
It's now pointing to routes.

I also confirmed that the routes file has a "next step" in it:

> # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html